lib/exceptions.js

```diff
@@ -1,4 +1,4 @@
-var records = require('./exceptions.json');
+var records = require('./exceptions');
```

Comment:
You're right. It should load that one. I think it is *not* loading that one in this case because the loader is `exceptions.js` itself, so it detects the infinite loop and loads the `.json` instead (this particular casuistic [is hinted](https://nodejs.org/dist/latest-v6.x/docs/api/modules.html#modules_cycles), but not properly documented&nbsp;&mdash;&nbsp;or I can't find it).